### DISCUSSION TITLE
nvim-treesitter[latex]: Error during "tree-sitter generate" Volta error: Could not locate executable `tree-sitter` in your project.                                                                                                                                                                 Please ensure that all project dependencies are installed with `npm install` or `yarn install`

### DISCUSSION THREAD

**Discussion by imac8t** [ayamir/nvimdots#1380] [Q&A]
nvim-treesitter[latex]: Error during "tree-sitter generate"
Volta error: Could not locate executable `tree-sitter` in your project.                                                                                                                                                                 Please ensure that all project dependencies are installed with `npm install` or `yarn install`

**Comment by charliie-dev** [comment]
https://github.com/ayamir/nvimdots/wiki/Prerequisites#optional-packages
